packages/server/graphql/private/mutations/updateEmail.ts

```diff
@@ -10,19 +10,35 @@ const updateEmail: MutationResolvers['updateEmail'] = async (_source, {oldEmail,
     throw new Error('New email is the same as the old one')
   }
 
-  const user = await getUserByEmail(oldEmail)
+  const user = await pg
+    .selectFrom('User')
+    .selectAll()
+    .where('email', '=', oldEmail)
+    .executeTakeFirst()
+
   if (!user) {
     throw new Error(`User with ${oldEmail} not found`)
   }
 
-  // RESOLUTION
+  //Use identities array for the user above
+  const identitiesArray = user.identities
+
+  // Update the identities array for AuthIdentityLocal, isEmailVerified=false
```

Comment:
+1 There is no point in repeating the logic in a comment. It's ok to have a block comment for structuring, or even better to state the intent of the change, so something like
```
// unset isEmailVerified
```
or
```
// changing the email means it's no longer verified
```
would be better, but if in doubt, omit the comment